openshift_scalability/scripts/ptibenchmark.sh

```diff
@@ -0,0 +1,20 @@
+# The following script will execture A B Benchmark for pti(spectre).
+
+#Usage:
+#	nohup ./ptibenchmark.sh 2>&1 &
+
+#TODO: Abstract this script to have an A B node vertical concept, but do some changes in between the runs.
+
+#Optimize the register script to run this command there.
+for h in `oc get no |grep -v NAME|awk '{print $1}'`; do ssh $h "echo "export LC_CTYPE=en_US.UTF-8 >> ~/.bashrc; export LC_ALL=en_US.UTF-8 >> ~/.bashrc; pbench-register-tool-set; done
+
+#PTI Enabled
+for h in `oc get no |grep -v NAME|awk '{print $1}'`; do ssh $h "echo 1 > /sys/kernel/debug/x86/pti_enabled"; done 
+pbench-user-benchmark ~/svt/openshift_scalability/./nodeVertical.sh ptibench python > /tmp/n.log
```

Comment:
Can we pass the "python" argument for nodeVertical.sh script as command line arg to the ptibenchmark.sh script, so we can run either "python" or "golang" version of cluster loader

Reply:
sure, we'll abstract it more.